cosmos/operators/local.py

```diff
@@ -51,18 +51,17 @@
 
 from sqlalchemy.orm import Session
 
+import cosmos.dbt.runner as dbt_runner
 from cosmos.config import ProfileConfig
 from cosmos.constants import (
     OPENLINEAGE_PRODUCER,
 )
 from cosmos.dbt.parser.output import (
-    extract_dbt_runner_issues,
```

Comment:
Can we mark this method as deprecated and log a message saying that it will be removed in 2.0.0 ?

Reply:
Great suggestion; I did this in d921ed9ae077f61719b03375edffe4b586b9c5d1